docs/wiki/deployment/aws-logging.md

```diff
@@ -11,6 +11,10 @@ Some configuration is shared between the two plugins:
 --aws_profile_name VALUE                AWS config profile to use for auth and region config
 --aws_region VALUE                      AWS region override
 --aws_secret_access_key VALUE           AWS secret access key override
+--aws_sts_arn_role VALUE                AWS STS assume role arn
+--aws_sts_region VALUE                  AWS STS assume role region
```

Comment:
@theopolis ya I just tested this.  The sts region flag is needed.  When my sts assume role is in another region than the region I used for initial login.  As I mentioned above, this is the case for my setup most of the time.